src/clockcache.h

```diff
@@ -139,15 +139,17 @@ struct clockcache {
 
    // Stats
    cache_stats stats[MAX_THREADS];
+   size_t      lookup_size;
+   size_t      entry_size;
+   size_t      pincount_size;
+   size_t      batch_busy_size;
```

Comment:
Here is an example of a structure which has 4 member ptr-fields for which we have to allocate separate memory fragments.

Each of these memory fragment's size has to be independently tracked. So, it is difficult to have just one structure-wide `mf_size` field to track size of fragment allocation.

(This point is in continuation of the [alternate solution remark](https://github.com/vmware/splinterdb/pull/569/files#r1421429375) on L1837 above.)